packages/api/AlvTime.Business/InvoiceRate/InvoiceRateService.cs

```diff
@@ -52,6 +55,89 @@ public async Task<decimal> GetEmployeeInvoiceRateForPeriod(DateTime fromDate, Da
         return billableHours / availableHours;
     }
 
+    public async Task<InvoiceStatisticsDto> GetEmployeeInvoiceStatisticsByMonth(DateTime fromDate, DateTime toDate)
+    {
+        var user = await _userContext.GetCurrentUser();
+        var invoiceRatePeriodStart = GetUserStartDateOrFromDate(fromDate, user);
+        invoiceRatePeriodStart = new DateTime(invoiceRatePeriodStart.Year, invoiceRatePeriodStart.Month, 1);
```

Comment:
Kunne ikke egentlig funksjonen GetUserStartDateOrFromDate bare returnert korrekt dato med en gang i stedet for å gjøre det over to linjer?

Reply:
Den funksjonen brukes også til å regne ut raten, så den funksjonen blir brukt to plasser. Om funksjonen bare ble brukt her så hadde jeg vært enig :+1: